packages/scxa-cell-type-wheel-experiment-heatmap/src/CellTypeWheelExperimentHeatmap.js

```diff
@@ -48,13 +48,18 @@ function CellTypeWheelExperimentHeatmap(props) {
         defaultValue={ { term: props.searchTerm, category: `metadata` } }
         enableSpeciesSelect={true}
         speciesSelectClassName={`small-4 columns`}
+        defaultSpecies={props.species}
       />
       <div className={`row-expanded small-12 columns`}>
         <div className={`small-12 medium-6 columns`} aria-label={`Cell type wheel`}>
           {props.searchTerm.trim() ?
             <CellTypeWheelFetchLoader
               host={props.host}
-              resource={URI(props.searchTerm, props.cellTypeWheelResource).toString()}
+              resource={URI(props.cellTypeWheelResource)
+                .segment(props.searchTerm)
+                .search(`?species=` + props.species)
```

Comment:
`props.species` only works for the first landing page, but if you change the `species` from the dropdown menu, the search term should change too. Please fix the logic here. Thanks.

Reply:
@lingyun1010 As we discussed it in our status meeting this case is covered in this ticket: [Species filter is getting ignored in the metadata search](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/426)
This current fix is for this ticket: [Species parameter is not passed from homepage](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/430)